porcelain/miner.go

```diff
@@ -509,3 +509,57 @@ func MinerGetCollateral(ctx context.Context, plumbing mgaAPI, minerAddr address.
 	}
 	return types.NewAttoFILFromBytes(rets[0]), nil
 }
+
+// mwapi is the subset of the plumbing.API that Miner(Set|Get)WorkerAddress use.
```

Comment:
I would either move `MinerGetWorker` to use this interface, or else remove the reference to Get here.